### PR TITLE
Labyrinth - Arianna & Sara

### PR DESCRIPTION
**Pull request by ariallahyar**
This project was done in collaboration between Arianna and Sara. Link to live project: https://salabyrinth.netlify.app/

### CODE REVIEW

**Review by JaEngd — APPROVED** [review]
Hi!

You've done a great job creating this project! Your code is clean, readable and you also managed to complete some stretch goals.

I really enjoyed that you made the game more user friendly by structuring the text bits in a clear way and by showing the disabled and enabled buttons. As well as displaying the history at the end. 

It's been inspiring looking at your solutions and your way of structuring your project. I'll look through the code a bit closer to see if I can add some useful comments.

All the best,
Jacob

File: code/src/components/DirectionButtons.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    state.gameplay.question.actions?.map((item) => item.direction)
+  );
+  const dispatch = useDispatch();
+
+  const direction = {
+    north: "North",
+    west: "West",
+    east: "East",
+    south: "South",
+  };
```
Comment (jenniferwvng) [inline code/src/components/DirectionButtons.js:51]: I like the logic here with the buttons and simplifies readability with having it stored as an object as well

File: code/src/components/Start.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        <Input
+          id="name"
+          type="text"
+          value={inputValue}
+          required
+          autofocus
+          onChange={(event) => setInputValue(event.target.value)}
+          placeholder="Player name..."
+        />
+        <Button type="submit" disabled={inputValue.length === 0}>
```
Comment (jenniferwvng) [inline code/src/components/Start.js:60]: Nice!

File: code/src/components/Summary.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+const Summary = () => {
+  const finalDescription = useSelector((state) => state.gameplay.question.description);
+  const moves = useSelector((state) => state.gameplay.moves);
+  const dispatch = useDispatch();
+
+  return (
+    <>
+      <Header>{finalDescription}</Header>
+      <h3>Your path took {moves.length} {moves.length === 1 ? "move:" : "moves:"}</h3>
```
Comment (jenniferwvng) [inline code/src/components/Summary.js:45]: Nice

**Pull request closed without merging**